src/search_renderer.cpp

```diff
@@ -98,9 +98,11 @@ std::string SearchRenderer::getHtml()
     std::string zim_id(it.getZimId());
     result.set("resultContentId", mp_nameMapper->getNameForId(zim_id));
     std::shared_ptr<zim::Archive> archive;
-    try {
+    if (!mp_library) {
+      result.set("bookTitle", kainjow::mustache::data(false));
```

Comment:
Wouldn't the same effect be achieved by *not* setting "bookTitle" at all?

Reply:
I don't know how mustache behave in case of absence of variable.
So I prefer to add to false. And it make more sense as we use it if the template (at least for testing it)